sdks/java/core/src/main/java/org/apache/beam/sdk/testing/PAssert.java

```diff
@@ -17,8 +17,9 @@
  */
 package org.apache.beam.sdk.testing;
 
+import static org.apache.beam.sdk.testing.PAssert.FAILURE_COUNTER;
```

Comment:
I don't really understand these imports, or why checkstyle is OK with them. Am I missing something?